<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="DynamicLogo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Assets/Logo/Music.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="MainLogo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Assets/Logo/JulleeOFF.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Debug_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DEBUG: Move Main Logo to side</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Debug_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DEBUG: Move Main Logo to Center</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Debug_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DEBUG: Activate ON/OFF</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Debug: Logo Change</string>
    </property>
    <widget class="QPushButton" name="Button_Youtube">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Youtube</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Wiki">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Wiki</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_News">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>News</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Who">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Who</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_FB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>FB</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Music">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Music</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Google">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Google</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Weather">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Weather</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Time">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Debug_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>450</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DEBUG: Run Text</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::FixedPixelWidth</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="tabStopWidth">
     <number>81</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_mic">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Assets/Logo/Mic.png</normaloff>
      <normalon>../Assets/Logo/Mic_On.png</normalon>../Assets/Logo/Mic.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::FixedPixelWidth</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="tabStopWidth">
     <number>81</number>
    </property>
   </widget>
   <widget class="QLabel" name="Text_label1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>JULLEE</string>
    </property>
   </widget>
   <widget class="QLabel" name="Text_label2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 255);</string>
    </property>
    <property name="text">
     <string>Voice Recognition</string>
    </property>
   </widget>
   <zorder>Debug_1</zorder>
   <zorder>DynamicLogo</zorder>
   <zorder>MainLogo</zorder>
   <zorder>Debug_2</zorder>
   <zorder>Debug_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>Debug_4</zorder>
   <zorder>TextBox</zorder>
   <zorder>Button_mic</zorder>
   <zorder>TextBox_2</zorder>
   <zorder>Text_label1</zorder>
   <zorder>Text_label2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMove_to_Corner">
   <property name="text">
    <string>Move to Corner</string>
   </property>
   <property name="toolTip">
    <string>Move Item to Corner</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Assets/Jullee.qrc"/>
 </resources>
 <connections/>
</ui>
